docs/OnTrack/Jupyter Notebook/Meeting Minutes/2.5.22.md

```diff
@@ -0,0 +1,95 @@
+# Jupyter Notebook - Meeting Minutes
+
+- [Back to Jupyter Notebook Documentation Index](../Index.md)
+
+### Date/Time of Meeting: 2nd May 2022, 6pm
+
+### Attendees: Ethan K, Lachlan F, Jordan L, Matt C, Chetan N
+
+### Minutes by: Matt C (with input from Lachlan F, Ethan K and Jordan L)
+
+### Next Meeting: 9th May
```

Comment:
I'm not sure if this is from the template or not, but a suggestion could be to make this normal text with a heading such as "Admin" (I don't have a better name off the top of my head).

This will draw the eye to the content rather than the less important info. Headings stand out a lot so you don't want them to overpower 👍